cmd/options/options.go

```diff
@@ -88,6 +94,11 @@ func (npdo *NodeProblemDetectorOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&npdo.ServerAddress, "address",
 		"127.0.0.1", "The address to bind the node problem detector server.")
 
+	fs.IntVar(&npdo.PrometheusServerPort, "prometheus-port",
+		20257, "The port to bind the Prometheus scrape endpoint. Use 0 to disable.")
```

Comment:
Update the description to explicitly state prometheus exporter is enabled by default, and use 0 to disable.

Reply:
Done.